pkgs/package-overrides.nix

```
@@ -356,5 +356,17 @@ in
           )
           (attrs.patches or []);
     });
+
+    gd =
+      if lib.versionOlder prev.php.version "7.4" then
+        prev.extensions.gd.overrideAttrs (attrs: {
+          buildInputs = attrs.buildInputs ++ [
+            pkgs.zlib
```

Would be nice to have an explanation.

```suggestion
            # Older versions of PHP check for these libraries even when not using bundled gd.
            pkgs.zlib
```